src/accountlist/accountlistSelectors.js

```diff
@@ -0,0 +1,22 @@
+import { fromJS } from 'immutable';
+
+export const getActiveAccount = (state) =>
+  state.accountlist.get(0);
+
+export const getAuth = (state) => {
+  const account = getActiveAccount(state);
+
+  if (!account) {
+    return fromJS({
```

Comment:
No special reason, they are identical. Often is shorter and does deep conversion, not just one level. I am open to changing it at later stage.